README.md

```diff
@@ -29,10 +36,19 @@ android_library(
 )
 ```
 
-You can see the full list of generated targets in [gmaven.bzl](https://raw.githubusercontent.com/aj-michael/gmaven_rules/master/gmaven.bzl).
+You can see the full list of generated targets in [`gmaven.bzl`](https://raw.githubusercontent.com/aj-michael/gmaven_rules/master/gmaven.bzl).
 
-To regenerate gmaven.bzl, run the following command. It will take about 5 minutes.
+# Regenerating targets
+
+To regenerate `gmaven.bzl`, run the following command. It will take about 5 minutes.
 
 ```
 rm gmaven.bzl && javac GMavenToBazel.java && java GMavenToBazel
 ```
+
+
+# Known issues
+
+Some of the artifacts depend on other artifacts that are not present on Google
+Maven. These targets do not work as the cross-repository resolution is not
```

Comment:
Perhaps mention something like "These dependencies are silently ignored and may cause failures at runtime."

Reply:
done